### ISSUE TITLE
adding a className breaks JSX compile

### ISSUE THREAD

**Issue by Pomax** [paramaggarwal/react-dropzone#9]
for some reason if I use `<Dropzone .../>` without a className, everything works fine, but adding my own `<Dropzone className="dropzone" ... />` breaks JSX compile because of
```
Error: Invariant Violation: The `style` prop expects a mapping from style properties to values, not a string.
```
You'll have to use something like this:
```
...
var className = "dropzone",
    style = DropzoneStyle;

if (this.props.className) {
  className = this.props.className;
  style = false;
}

return (<div ... className={className} style={style} ...>
  ...
</div>);
```

**Comment by paramaggarwal** [comment]
Currently you can style it by passing a `style` prop.

**Comment by Pomax** [comment]
I'm sure, but if I want to rely on CSS because of automation (LESS compiled CSS, for instance), that is not a solution =)

**Comment by paramaggarwal** [comment]
That's right. Will keep this open and fix it sometime.

**Comment by Pomax** [comment]
I filed a PR with some tweaks that I found critically missing over on https://github.com/paramaggarwal/react-dropzone/pull/10, which also fixes this issue

**Comment by paramaggarwal** [comment]
Thanks! Merged your PR, published as v1.0.1 (https://github.com/paramaggarwal/react-dropzone/releases/tag/v1.0.1)
